### PR TITLE
Add Chloe's profile

### PR DESCRIPTION
**Pull request by Aibono1225**

### CODE REVIEW

File: Chloe-Lin/Chloe-Lin.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+id: "6ed524e2-8933-eb11-a813-000d3a79983b&viewType=1039"
+
+
+quote: "There were many ways down Mount Fuji, according to my guidebook, but only one way up. Life lesson in that, I thought."
+quoteAuthor: "Phil Knight"
+---
+
+[Editing your profile]: https://github.com/SSWConsulting/People/wiki/3.-Editing-your-profile
```
Comment (tiagov8) [inline Chloe-Lin/Chloe-Lin.md:9]: This shouldn't be here @Aibono1225 ;)

File: Chloe-Lin/Chloe-Lin.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+id: "6ed524e2-8933-eb11-a813-000d3a79983b&viewType=1039"
+
```
Comment (tiagov8) [inline Chloe-Lin/Chloe-Lin.md:3]: any qualifications to be added here?

Comment (Aibono1225) [reply Chloe-Lin/Chloe-Lin.md:3]: No I don't have any qualifications for now 😢

File: Chloe-Lin/Chloe-Lin.md (hunk 1 above)
Comment (Aibono1225) [reply Chloe-Lin/Chloe-Lin.md]: Done - I deleted it :)

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**